openassessment/xblock/static/dist/manifest.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Should we be including this manifest file or is this just a developer file we should `.gitignore` out of GitHub?

Reply:
I agree that it is generated. However, it is entangle with javascript translation. For now,  it is better just to leave it in.